.github/ISSUE_TEMPLATE/Doc Report.yml

```diff
@@ -0,0 +1,61 @@
+name: Docs Report
+description: Help us improve our docs
+title: "docs: "
+labels: ["documentation"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this report!
+
+  - type: textarea
+    id: current-behaviour
+    attributes:
+      label: Current Behaviour
+      description: Tell us what could be improved.
+      placeholder: A typo was encountered at line X!
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected-behaviour
+    attributes:
+      label: Expected Behaviour
+      description: How should we improve?
+      placeholder: Fix the typo at line X!
+
+  - type: dropdown
+    id: browsers
+    attributes:
+      label: What browsers are you seeing the problem on?
+      multiple: true
+      options:
+        - Firefox
+        - Chrome
+        - Safari
+        - Microsoft Edge
+        - Other
+
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: shell
+
+  - type: textarea
+    id: additional-context
+    attributes:
+      label: Additional Context
+      description: Anything else that you want to share?
+      placeholder: The change suggested is important because...
+
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
```

Comment:
@bamblebam You could probably add one more checkbox for Contributing guidelines other than that everything looks good.

Reply:
I will just modify it to say code of conduct and contributing guidelines

Reply:
Done

Reply:
Yeah looks good.